src/components/EventMap.js

```diff
@@ -1,114 +1,100 @@
 import Vue from 'vue';
-import { pollForNewEvents, loadEvents } from 'src/util/events-poll';
+import { getFilteredEvents } from 'src/util/events';
+
+// Attaches L (leaflet.js) to window.
+import 'mapbox.js';
```

Comment:
I think the code splitting is cool and could be promising but overall I don't think it was giving us much in this case, loading mapbox asynchronously, cause we really want mapbox as soon as possible to be able to render a map while the larger data payloads are arriving asynchronously.